### PR TITLE
Feature/organize css iteration0

### PR DESCRIPTION
**Pull request by lilianaweimer**
**Organized CSS rules into categories based on grid**
- Discussed how to organize rules previously with group members
- Inside of each category, organized rules based on specificity and appearance in HTML

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -8,6 +10,18 @@ body {
...
+  padding-left: 15px;
+  padding-right: 15px;
+}
+
+input {
+  border-color: #5E60A9;
+  border-radius: 2px;
+}
+
+/* IDEABOX RULES (SIDEBAR) */
```
Comment (jennadilullo8) [inline styles.css:23]: I love that each section is categorized!

File: styles.css [hunk 2]
```diff
@@ -65,12 +65,6 @@ button.show-starred {
   justify-content: space-evenly;
 }
 
-.search-bar {
```
Comment (jennadilullo8) [inline styles.css:89]: The ideabox looks so much cleaner!

File: styles.css [hunk 3]
```diff
@@ -120,6 +120,13 @@ button.save {
   justify-content: space-between;
 }
 
+.star-x-button {
```
Comment (jennadilullo8) [inline styles.css:123]: I love how everything looks!

File: styles.css (hunk 1 above)
Comment (asodemann18) [reply styles.css:23]: Looks great!

**Pull request merged**